doc/source/whatsnew/v2.0.1.rst

```diff
@@ -21,6 +21,7 @@ Fixed regressions
 - Fixed regression in :meth:`MultiIndex.isin` raising ``TypeError`` for ``Generator`` (:issue:`52568`)
 - Fixed regression in :meth:`Series.describe` showing ``RuntimeWarning`` for extension dtype :class:`Series` with one element (:issue:`52515`)
 - Fixed regression when adding a new column to a :class:`DataFrame` when the :attr:`DataFrame.columns` was a :class:`RangeIndex` and the new key was hashable but not a scalar (:issue:`52652`)
+- Regression with Bug in DataFrame.to_string() creates extra space for string dtypes in pandas >=2.0 (:issue:`52690`)
```

Comment:
```suggestion
- Fixed regression in :meth:`DataFrame.to_string` creates extra space for string dtypes (:issue:`52690`)
```

Could you also move this to `v2.0.1.rst`?

Reply:
yep agreed

Reply:
Ok did this